ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -736,9 +736,25 @@ objects:
         fi
 
         # TODO: Replace with a more concise manifest injection approach
-        if [[ -n "${CLUSTER_NETWORK_MANIFEST:-}" ]]; then
+        if [[ -n "${CLUSTER_NETWORK_MANIFEST:-}" ]] || has_variant "rt"; then
             openshift-install --dir=/tmp/artifacts/installer/ create manifests
-            echo "${CLUSTER_NETWORK_MANIFEST}" > /tmp/artifacts/installer/manifests/cluster-network-03-config.yml
+
```

Comment:
nit: this should probably be flattened out:
```
if [[ -n "${CLUSTER_NETWORK_MANIFEST:-}" ]] || has_variant "rt"; then
    openshift-install --dir=/tmp/artifacts/installer/ create manifests
fi
if [[ -n "${CLUSTER_NETWORK_MANIFEST:-}" ]]; then
    echo ...
fi
if has_variant "rt"; then
    cat > ...
fi
```
or at least untangled:
```
if [[ -n "${CLUSTER_NETWORK_MANIFEST:-}" ]]; then
    openshift-install...
    echo ...
fi
if has_variant "rt"; then
    openshift-install...
    cat >...
fi
```

Reply:
Thanks for your review!

About your 1st suggestion: not sure, doesn't it make more sense to deal with both features which need to modify manifests inside the `if` branch which creates them?

About the 2nd: that might probably break if both features are used simultaneously, it would call `create manifests` twice.

Reply:
Yes, keep the variant separate.  CLUSTER_NETWORK_MANIFEST is a highly specialized use case and support is limited.

Reply:
split them, and then insight has_variant "rt" have an error condition that fails if CLUSTER_NETWORK_MANIFEST is set:

```
if has_variant "rt"; then
  if [[ -n "{CLUSTER_NETWORK_MANIFEST:-}" ]]; then
    echo 'error: CLUSTER_NETWORK_MANIFEST is incompatible with the `rt` variant'
    exit 1
  fi
```

Reply:
ok, done